upgrade.sh

```diff
@@ -24,6 +26,12 @@ if [ "$EUID" -eq 0 ]; then
     exit 1
 fi
 
+# Windows Git Bash docker exec compatibility fix
+if type winpty > /dev/null 2>&1; then
+    shopt -s expand_aliases
+    alias docker="winpty -Xallow-non-tty -Xplain docker"
```

Comment:
I'm glad you discovered/added this!

Reply:
Results of my adventure down the Windows rabbit hole....